sdk/python/lib/pulumi/run/plugin/__main__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2025, Pulumi Corporation.
```

Comment:
I think we could also move `pulumi-language-python-exec` to `pulumi.run.program`.